src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+
+public class AdsVelesObject extends AdsBaseObject {
+
+  private static final String INCONTENT_VIDEO = ".video-display-wrapper";
+
+  public static final String SECOND_ARTICLE_HEADER_OASIS = "#Player_above";
+  public static final String FIRST_ARTICLE_HEADER_MOBILE = "#Header";
+
+  public final String BIDDER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_fallback_bid.*";
+  public final String DIRECT_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_direct.*";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "DIRECT_PLAYER_EVENT_PATTERN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)